app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/LoginActivity.java

```diff
@@ -32,6 +35,8 @@
 
     String username, password;
     ReaderApi api;
+    @Inject
+    MockServer mockServer;
```

Comment:
@Anna812 the point would be to inject the ReaderApiInterface so the usage will be the same and this class won't change, you just change the the implementation of it